reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -1,10 +1,12 @@
 """Setup for signac, signac-flow, signac-dashboard for this study."""
 # import foyer
 import os
-import pathlib
+from pathlib import Path
 
 import flow
-from flow import environments
+from flow import directives
+
+import reproducibility_project.templates.ndcrc
```

Comment:
This part is a bit out of my current domain knowledge. But do you need to do anything like instantiate the `reproducibility_project.templates.ndcrc.NotreDameCRC` object in your project.py? Or does this just work for you?

Reply:
Nope, it just works.  This is apparently because `ComputeEnvironment` is a meta-class, according to https://docs.signac.io/en/latest/environments.html#how-to-use-environments .

Reply:
That is awesome, and I learned something new!